uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/AbsencePeriodTest.php

```diff
@@ -72,8 +72,8 @@ public function testIfWeightIsEmptyItWillBeMaxWeightPlusOne()
   }
 
   /**
-   * @expectedException PEAR_Exception
-   * @expectedExceptionMessage DB Error: already exists
+   * @expectedException CRM_HRLeaveAndAbsences_Exception_InvalidAbsencePeriodException
```

Comment:
The alias wasn't working on the tests. I've not tried it before anyway. 
Seems phpunit needs the fully qualified class name.